pynecone/components/base/link.py

```diff
@@ -19,3 +21,33 @@ def _render(self) -> Tag:
             href=self.href,
             rel=self.rel,
         )
+
+class ScriptTag(Component):
+    """A component that creates a script tag with the speacified type and source."""
+
+    type: Var[str]
+
+    source: Var[str]
+
+    integrity: Optional[Var[str]] = None
```

Comment:
We don't need default values, they will automatically be set to `None`

Reply:
Is the source correct? Could it be src ?